ui/src/singletons/store.ts

```diff
@@ -195,6 +206,49 @@ const useStore = create<AppStore>()(
             console.error("Failed to import an EIP flow JSON. Malformed input")
             return {}
           }),
+
+        updateLayoutOrientation: (orientation: Layout["orientation"]) =>
+          set((state) => {
+            const newLayout: Layout = {
+              orientation: orientation,
+              density: state.layout.density,
+            }
```

Comment:
:memo: To make this a bit more robust, you can use the spread operator:
```
const newLayout: Layout = {
  ...state.layout,
  orientation: orientation,
}
```
Then you don't have to worry about changing it if other fields were added to the layout object. This change can also be made in `updateLayoutDensity`.